internal/peer/channel/channel.go

```diff
@@ -86,6 +89,7 @@ func resetFlags() {
 	flags.StringVarP(&outputBlock, "outputBlock", "", common.UndefinedParamValue, `The path to write the genesis block for the channel. (default ./<channelID>.block)`)
 	flags.DurationVarP(&timeout, "timeout", "t", 10*time.Second, "Channel creation timeout")
 	flags.BoolVarP(&bestEffort, "bestEffort", "", false, "Whether fetch requests should ignore errors and return blocks on a best effort basis")
+	flags.DurationVarP(&tlsHandshakeTimeShift, "tlsHandshakeTimeShift", "", 0, "The amount of time to shift backwards during TLS handshakes")
```

Comment:
Perhaps expand a bit in the descriptive text, something like "... of time to shift backwards for certificate expiration checks during ..."

Reply:
Done